### ISSUE TITLE
Audio not Working / Cannot update Crouton

### ISSUE THREAD

**Issue by dahveed** [dnschneid/crouton#2174]
Hi,

My audio has stopped working in the chroot after I updated ChromeOS. I tried searching and reading posts here about audio failing, but I got muddled as to what to do. 

Several threads state updating crouton will do the trick, and so that's what I tried. Here's what happened:

chronos@localhost / $ sudo sh ~/Downloads/crouton -u
Password: 
/usr/local/chroots/precise does not exist; cannot update.
Valid chroots:
trusty

chronos@localhost / $ sudo sh ~/Downloads/crouton -u -n trusty
/usr/local/chroots/trusty already exists; updating it...
Enter encryption passphrase for trusty: 
Unable to determine the release in /usr/local/chroots/trusty. Please specify it with -r.

chronos@localhost / $ sudo sh ~/Downloads/crouton -u -n trusty -r trusty
/usr/local/chroots/trusty already exists; updating it...
Enter encryption passphrase for trusty: 
Release doesn't match! Please correct the -r option, or specify a second -u to change the release, upgrading the chroot (dangerous).

chronos@localhost / $ sudo sh ~/Downloads/crouton -u -u -n trusty -r trusty
/usr/local/chroots/trusty already exists; updating it...
Enter encryption passphrase for trusty: 
WARNING: Changing the chroot release to trusty.
Press Control-C to abort; upgrade will continue in 5 seconds.
chronos@localhost / $ 

---------------

The above errors in Crosh seem the same as https://github.com/dnschneid/crouton/issues/1695 . But I don't know how to apply the "Master Git" version in that thread. 

The chroot still starts, but the audio is still broken.

Any help would be much appreciated!

**Comment by dahveed** [comment]
This is the closest thread to my problem  ( https://github.com/dnschneid/crouton/issues/2015 ) .  Alsamixer doesn't work, and Pulseaudio isn't working either (but it's installed in apt).

Since the last advice of the thread is to update crouton, that's what I tried. I thought I updated Crouton, but I discovered that it wasn't really updated. After forcing the update in Crosh with "double -u" ( crouton -u -u -r trusty), there's no error message like "update failed." However, when in the chroot (which opens fine, but there's no sound), the crouton version is old. I don't know why crouton says it cannot recognize my chroot release version, so I suspect this is part of the reason I can't update.

Here's my current (old) crouton version from terminal within the chroot:
 
crouton: version 1-20150620222151~master:89796a99
release: trusty
architecture: amd64
xmethod: xorg
targets: xfce
host: version 7262.57.0 (Official Build) stable-channel squawks 
kernel: Linux localhost 3.10.18 #1 SMP Thu Sep 17 23:07:25 PDT 2015 x86_64 x86_64 x86_64 GNU/Linux
freon: yes

Since updating crouton would seem to be the logical best step to fix my chroot audio problems, I'm hoping someone could help me do that!

**Comment by dahveed** [comment]
Update: after more fiddling and scouring the web and this wiki, I gave up on fixing the couton update issue. Instead, I created a new chroot and am currently setting it up in a manner similar to the old chroot. The new chroot's sound works just fine and seems to have no issues. 

Since this is roughly the equivalent of reinstalling your operating system, it's a bit inconvenient, but it's also been an opportunity to look at the bloat I had installed in the old chroot. As soon as I set it up the way I like and migrate all the files, I'll delete the old chroot.
